CHANGELOG.md

```diff
@@ -14,7 +14,6 @@
 
 ### Changed
 
-- Use kubespray repository
```

Comment:
Why is this removed?

Reply:
We switched back to our own repo, for some fixes. But maybe i should keep that and add a row that we changed back?

Reply:
Riight... Well hmmm, yea then I think that I like this instead of having both added and reverted in the same release notes